pokemongo_bot/test/step_walker_test.py

```diff
@@ -91,17 +107,95 @@ def test_small_distance_small_step(self):
         self.bot.config.walk_max = 1
         self.bot.config.walk_min = 1
 
-        sw = StepWalker(self.bot, 1e-5, 1e-5)
-        self.assertEqual(sw.dLat, 0)
-        self.assertEqual(sw.dLng, 0)
+        # distance from origin 0,0 to 1e-6, 1e-6 is 0.157253373328 meters
+        total_distance = Geodesic.WGS84.Inverse(0.0, 0.0, 1e-6, 1e-6)["s12"]
```

Comment:
if you used geodesic in StepWalker... use haversine in tests for distance!

in the end this is how we catch differences